src/shogun/base/SGObject.cpp

```diff
@@ -1059,6 +1072,14 @@ CSGObject* CSGObject::create_empty() const
 	return object;
 }
 
+void CSGObject::initialise_auto_params()
+{
+	auto params = self->filter(ParameterProperties::AUTO);
+	for (const auto& param: params) {
```

Comment:
style problem... but i wonder why the clang-format job didn't catch this... need to check